pkg/workspaces/workspaces.go

```diff
@@ -16,30 +16,35 @@ package workspaces
 
 import (
 	"errors"
+	"fmt"
 	"strings"
 
+	"github.com/tektoncd/cli/pkg/cli"
```

Comment:
nit: I would love to see those packages (`pkg/workspaces`, `pkg/file`) to not depend on `pkg/cli` but it seems to be a way bigger refactoring than that PR should change :sweat_smile:

Reply:
As a follow up from this pr, maybe we could explore an interface option here to fix packages that depend on `pkg/cli`? I think there are others at the `pkg` level as well.

Reply:
yeah :)